core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -109,288 +178,224 @@ public void setId(long id) {
     this.projectId = id;
   }
 
+  @Override
   @Deprecated
   public Long getProjectId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 172. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)